README.md

```diff
@@ -222,11 +222,12 @@ If your macOS application supports macOS 10.14 or prior, you must run `myValet.m
 Valet guarantees it will never fail to write to or read from the keychain unless `canAccessKeychain()` returns `false`. There are only a few cases that can lead to the keychain being inaccessible:
```

Comment:
nit: Maybe update this to "... will never fail to write _valid data_ to or read from..." to indicate that there are certain data inputs where it still might fail?

Reply:
Solid call. This also covers the case where someone tries to write an empty string to the keychain.
```suggestion
Valet guarantees it will never fail to read from or write valid data to the keychain unless `canAccessKeychain()` returns `false`. There are only a few cases that can lead to the keychain being inaccessible:
```